html/audiobridgetest.js

```diff
@@ -298,8 +298,12 @@ $(document).ready(function() {
 									$('#room').removeClass('hide').show();
 									$('#participant').removeClass('hide').html(myusername).show();
 								},
-								onremotetrack: function(track, mid, on) {
-									Janus.debug("Remote track (mid=" + mid + ") " + (on ? "added" : "removed") + ":", track);
+								onremotetrack: function(track, mid, on, metadata) {
+									Janus.debug(
+										"Remote track (mid=" + mid + ") " +
+										(on ? "added" : "removed") +
+										(metadata ? " (" + metadata.reason + ") ": "") + ":", track
```

Comment:
Just a code style nit: please put an extra space before the `:` that belongs to the ternary operator, as it's not clear otherwise. The same nit applies to a few other instances I see you've added.